OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalDbContract.java

```diff
@@ -81,5 +81,6 @@
       public static final String COLUMN_NAME_DISPLAY_QUANTITY = "display_quantity";
       public static final String COLUMN_NAME_LAST_DISPLAY = "last_display";
       public static final String COLUMN_CLICK_IDS = "click_ids";
+      public static final String COLUMN_DISPLAYED = "displayed";
```

Comment:
1. `COLUMN_DISPLAYED` should be `COLUMN_DISPLAYED_IN_SESSION`
2. `displayed` should be `displayed_in_session`